website/docs/api-layers.md

```diff
@@ -1531,6 +1531,43 @@ model.initialize()
 https://github.com/explosion/thinc/blob/master/thinc/layers/with_nvtx_range.py
 ```
 
+### with_signpost_interval {#with_signpost_interval tag="function"}
+
+<inline-list>
+
+- **Input:** <tt>Any</tt>
+- **Output:** <tt>Any</tt>
+
+</inline-list>
+
+Layer that wraps any layer and marks the init, forward and backprop passes as a
+(macOS) signpost interval. This can be helpful when profiling the performance of
+a layer using macOS Instruments.app. Use of this layer requies that the
+`os-signpost` package is installed.
```

Comment:
Shall we link out to https://github.com/explosion/os-signpost?

Reply:
Added, I also added a link for Instruments.app.